pkg/deps/maven.go

```diff
@@ -0,0 +1,162 @@
+package deps
+
+import (
+	"archive/zip"
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+
+	"github.com/apache/skywalking-eyes/license-eye/internal/logger"
+	"github.com/apache/skywalking-eyes/license-eye/pkg/license"
+)
+
+func init() {
+	var err error
+	_, err = exec.Command("mvn", "--version").Output()
+	if err != nil {
+		return
+	}
+
+	Resolvers = append(Resolvers, new(MavenPomResolver))
```

Comment:
I think this is not a good condition to check whether we should add the resolver:

- `mvn version` is not enough to determine whether maven resolver can be use, [maven wrapper](https://github.com/takari/maven-wrapper) is also widely used in some projects.
- We should log some warning / error logs when users' environment don't have maven installed, instead of failing silently.

My suggestion is 

- just add the resolver to https://github.com/apache/skywalking-eyes/blob/d6e232b01042d112b103ae6c7f51c5c677e1f37e/pkg/deps/resolve.go#L29-L32
- check whether `mvnw` exists in the same directory as `pom.xml` first, if yes, use `mvnw`, otherwise, use `mvn`
- run the needed command (`mvn copy-dependencies`), if it failed, we also fail the command and print some logs